ui/src/app/views/homepage/component.tsx

```diff
@@ -334,8 +334,13 @@ export const Homepage = withUserProfile()(class extends React.Component<
           this.setState({firstVisitTraining: false});
         }
       }
-
-      this.setState({eraCommonsLinked: !!profile.linkedNihUsername});
+      try {
+        const syncEraCommonsStatus = await profileApi().syncEraCommonsStatus();
```

Comment:
does this call firecloud?  if not why not just use the `profileState` prop?

Reply:
It does, but I realized that we actually don't need to sync here if we update when the callback from NIH happens.